.github/workflows/scripts/pre-submit.apis/verify-safefs.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+set -euo pipefail
+
+# Verify that only safe file functions are used.
+results=$(
+    find .github/actions/ actions/ -maxdepth 3 -name '*.ts' -not -path "*/tscommon/*" -not -path "*/__tests__/*" -type f -print0 |
+    xargs -0 grep '"fs"' ||
```

Comment:
nit: only catch imports
```suggestion
    xargs -0 grep 'import.*"fs"' ||
```

Reply:
Updated to `xargs -0 grep -P '(import.*|require.*)"fs"'` to catch `require("fs")` too.